Source/WebCore/page/ResizeObservation.cpp

```diff
@@ -92,6 +92,9 @@ auto ResizeObservation::computeObservedSizes() const -> std::optional<BoxSizes>
 
 LayoutPoint ResizeObservation::computeTargetLocation() const
 {
+    if (!m_target)
```

Comment:
Hm... I think we're supposed to keep this element alive via `ResizeObserver::m_activeObservationTargets`. It's unfortunate that we don't have a reproduction for this crash.